cmd/minishift/cmd/profile/profile_set.go

```diff
@@ -26,9 +26,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const (
+/*const (
```

Comment:
-1, if something is not needed anymore, it should be removed. What's your intent with commenting this out?

Reply:
My bad, left it unknowingly